pkg/controllers/dashboard/helm/repo.go

```diff
@@ -90,11 +92,16 @@ func (r *repoHandler) ClusterRepoDownloadStatusHandler(repo *catalog.ClusterRepo
 		return status, nil
 	}
 
+	interval := 6 * time.Hour
```

Comment:
Maybe we can use the setting `SystemFeatureChartRefreshSeconds` as the default value?